labs/smartbft.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Hyperledger Fabric SmartBFT Library
+parent: Labs
+
+---
+# Lab Name
+
+[SmartBFT] - https://github.com/SmartBFT-Go/consensus
+
+
+# Short Description
+
+Hyperledger Fabric is an enterprise-grade, permissioned distributed ledger platform that offers modularity for a broad set of industry use cases. One modular component is a pluggable ordering service that establishes consensus on the order of transactions and batches them into blocks. This proposal suggests integrating the SmartBFT consensus library into Hyperledger Labs to enhance its blockchain frameworks with a robust, scalable, and resilient consensus mechanism designed for practical BFT deployment.
+
+This lab provides a BFT consensus library based on the [BFTSmart](https://github.com/bft-smart/library) protocol and written in Go. The library is tailored to the blockchain use case yet general enough to cater to a wide variety of other uses. The BFT library's design and integration into Fabric address crucial aspects that were left unsolved in all prior work, making it unfit for production use. 
```

Comment:
Can you clarify this part - "address crucial aspects that were left unsolved in all prior work, making it unfit for production use". What was the prior work? Why unfit for production use? How has it resolved?

Reply:
He is referring to previous works, and as to why it was unfit for production use, you can look [here](https://www.youtube.com/watch?v=DSJjI7AgFPA&t=770s).

Reply:
Thanks, it would still be good to update the .md accordingly @pfi79

Reply:
This paragraph refers to the previous work, aka:https://github.com/bft-smart/library, @denyeart. For the record, @jyellick and Kostas explained why the previous attempt to have SmartBFT into Fabric was unfitable for production. See here: https://lists.hyperledger.org/g/fabric/topic/17549966.

Reply:
fix